app/scripts/modules/amazon/src/loadBalancer/details/targetGroupDetails.html

```diff
@@ -91,7 +91,7 @@ <h3 class="horizontal middle space-between flex-1" select-on-dbl-click>
               <a ui-sref="^.serverGroup({region: serverGroup.region,
                                           accountId: serverGroup.account,
                                           serverGroup: serverGroup.name,
-                                          provider: 'aws'})">
+                                          provider: serverGroup.cloudProvider})">
```

Comment:
this link was also broken, so i fixed it 🤷‍♂️

Reply:
🥇